package.json

```diff
@@ -89,41 +89,41 @@
     "assert": "^2.1.0",
     "babel-loader": "^9.1.3",
     "babel-plugin-istanbul": "^7.0.0",
-    "bpmn-js": "^17.2.2",
-    "bpmn-js-create-append-anything": "^0.5.1",
-    "bpmn-js-properties-panel": "^5.14.0",
-    "bpmn-moddle": "^9.0.0",
-    "camunda-bpmn-js-behaviors": "^1.2.2",
+    "bpmn-js": "^17.9.1",
+    "bpmn-js-create-append-anything": "^0.5.2",
+    "bpmn-js-properties-panel": "^5.19.0",
+    "bpmn-moddle": "^9.0.1",
+    "camunda-bpmn-js-behaviors": "^1.4.0",
     "camunda-bpmn-moddle": "^7.0.1",
-    "chai": "^4.4.0",
-    "copy-webpack-plugin": "^12.0.0",
+    "chai": "^4.4.1",
+    "copy-webpack-plugin": "^12.0.2",
     "cross-env": "^7.0.3",
-    "diagram-js": "^14.4.0",
+    "diagram-js": "^14.8.0",
     "downloadjs": "^1.4.7",
-    "eslint": "^8.56.0",
-    "eslint-plugin-bpmn-io": "^1.0.0",
+    "eslint": "^8.57.0",
```

Comment:
We could already bump to `9`.

Reply:
Unfortunately not:

```
❯ npm install eslint@latest --save                                                                                                                              
npm WARN ERESOLVE overriding peer dependency                                                                                                                    
npm WARN While resolving: bpmn-js-element-templates@1.16.0                                                                                                      
npm WARN Found: eslint@8.57.0                                                   
npm WARN node_modules/eslint                                                                                                                                    
npm WARN   peer eslint@"^6.0.0 || ^7.0.0 || >=8.0.0" from @eslint-community/eslint-utils@4.4.0                        
npm WARN   node_modules/@eslint-community/eslint-utils                                                                                                          
npm WARN     @eslint-community/eslint-utils@"^4.2.0" from eslint@8.57.0                                                                                         
npm WARN   7 more (eslint-plugin-bpmn-io, eslint-plugin-import, ...)                                                                                            
npm WARN                                                                                                                                                        
npm WARN Could not resolve dependency:                                                                                                                          
npm WARN peer eslint@"^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 || ^8" from eslint-plugin-import@2.29.1                                                              
npm WARN node_modules/eslint-plugin-import                                      
npm WARN   dev eslint-plugin-import@"^2.29.1" from the root project                                                                                             
npm WARN ERESOLVE overriding peer dependency                                    
npm WARN While resolving: bpmn-js-element-templates@1.16.0                                                                                                      
npm WARN Found: eslint@8.57.0                                                   

```

Reply:
Oh OK, the import plugin is blocking: https://github.com/import-js/eslint-plugin-import/pull/2996